tests/15-rpl-classic/js/12-rpl-rank-error.js

```diff
@@ -0,0 +1,26 @@
+all_transmissions_received = false;
+
+function check_success() {
+  if(all_transmissions_received) {
+    log.testOK()
+  }
+  else {
+    log.testFailed()
+  }
+};
+
+TIMEOUT(600000, if(all_transmissions_received) { log.testOK(); });
+
+while(true) {
+  YIELD();
+
+  if(msg.contains("RPL Option Error: Dropping Packet")) {
+    log.log(time + " " + "node-" + id + " "+ msg + "\n");
+    log.testFailed();
+  }
+
+  else if(msg.contains("Received all packets")) {
+    log.log(time + " " + "node-" + id + " "+ msg + "\n");
+    all_transmissions_received = true
```

Comment:
Does this test always terminate with a timeout? Is there any reason to not let it terminate with testOK() here?

Reply:
This was intentional, but after review I can't really see the rationale - there is plenty of time after the nodes move to catch any issues. I have simplified the script.